<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{; /* Light gray background */
	background-color: rgb(35, 35, 35);
    border-radius: 10px;
color: rgb(255, 255, 255);}
QPushButton:hover{background-color: rgb(50,50,50); /* Light gray background */
    border-radius: 10px;
	color: rgb(255, 255, 255);
 border: 2px solid #ccc;}</string>
   </property>
   <property name="text">
    <string>Press to Continue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voidButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(54, 52, 59); /* Light gray background */
    border-radius: 10px;
color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>Welcome</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
